gearcmd/worker.go

```diff
@@ -43,6 +44,11 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 		data["error_message"] = err.Error()
 		data["value"] = 0
 		data["success"] = false
+		if conf.RetryCount > 0 {
+			log.Println(kayvee.FormatLog("gearcmd", kayvee.Error, "RETRY", data))
```

Comment:
I checked out the library and it looks nice.  But because we would have to implement another interface to use `backoff`, I think the code should be alright as it is now.  I'm definitely happy to change it though if that would be a better solution.  

Fixed the logs to just be `RETRY`.   :+1: